lib/service/Service.ts

```diff
@@ -144,26 +144,26 @@ class Service extends EventEmitter {
   }
 
   /**
-   * Get a list of standing peer orders from the order book for a specified trading pair, or for all
-   * trading pairs if no pair is specified.
+   * Get a list of standing orders from the order book for a specified trading pair.
    */
-  public getOrders = ({ pairId, maxResults }: { pairId?: string, maxResults?: number }) => {
-    const ret: OrderArrays = {
-      buyOrders: [],
-      sellOrders: [],
+  public getOrders = ({ pairId, maxResults }: { pairId: string, maxResults: number }) => {
+    checkArgument(pairId !== '', 'pairId must be specified');
+
+    const result: { [ type: string ]: OrderArrays } = {
+      peerOrders: {
```

Comment:
You can combine these lines with the ones below rather than initializing with empty arrays:

```javascript
const result = {
  peerOrders: this.orderBook.getPeerOrders(pairId, maxResults),
  ownOrders: this.orderBook.getOwnOrders(pairId, maxResults),
}
```